app/router.js

```diff
@@ -6,7 +6,11 @@ var Router = Ember.Router.extend({
 });
 
 Router.map(function() {
-  this.resource('login');
+  this.route('login');
+  this.resource('skills', function () {
+    this.route("featured");
+    this.route("all");
```

Comment:
Yes, but how do we get it to not display all skills then when we click on the featured skills, since featured is outputted from /skills/featured?